packages/core/src/modules/credentials/CredentialsModule.ts

```diff
@@ -305,6 +305,17 @@ export class CredentialsModule {
     }
   }
 
+  /**
+   * Declines an offer as holder
+   * @param credentialRecordId the id of the credential to be declined
+   * @returns credential record that was declined
+   */
+  public async declineOffer(credentialRecordId: string) {
+    const credentialRecord = await this.credentialService.getById(credentialRecordId)
+    await this.credentialService.declineOffer(credentialRecord)
```

Comment:
I think returning the return value of this function would be cleaner. However since in `this.credentialService.declineOffer(...)` we modify the parameter we receive it does not matter as much.